general-doc/decentralized-web.rst

```diff
@@ -19,25 +19,20 @@ Popular ETH smart contracts essentially follow that rule.
 
 However, most DeFi web3 apps such as `app.uniswap.org <https://app.uniswap.org/>`_ are
 typically used through an otherwise normal website that talks to a browser plugin
-(typically `Metamask <https://github.com/MetaMask/metamask-extension>`_) allowing the webpage
-to interact with the user's wallet and the blockchain. The website that serves as an
-entry point to the dApp is neither decentralized nor immutable-once-audited, thus breaking
-the very foundations of DeFi security. And that’s how you get into situations of
-scandalous thefts in the DeFi world, like
-`this one <https://www.theverge.com/2021/12/2/22814849/badgerdao-defi-120-million-hack-bitcoin-ethereum>`_.
+(typically `Metamask <https://github.com/MetaMask/metamask-extension>`_) allowing the webpage to interact with the user's wallet and the blockchain. The website that serves as an
+entry point to the dApp is neither decentralized nor immutable-once-audited.
+This breaks the very foundations of blockchain security. And that’s how you get into situations of scandalous thefts in the DeFi world, like `this one <https://www.theverge.com/2021/12/2/22814849/badgerdao-defi-120-million-hack-bitcoin-ethereum>`_.
 
 The goal here is to allow addresses on Massa to store not only a balance, bytecode and a
-datastore, but also files. Those files must remain small, or storing them will cost a lot to their owner.
```

Comment:
Maybe a 1-sentence explanation into what these are. To me it sounds like balance bytecode and datastore are  general knowledge and everyone everywhere knows about them, when in fact it's very specific to Massa, no?

Reply:
hmmm I probably lack of knowledge but I don't get what is not easy to understand... can you suggest sth to help me please? 
note: the audience will be developers so they understand the tech part.

Reply:
I was thinking to add a sentence to remind readers what: datastore and filestore are. But if you think it's clear, then ok. My point was that in order to read through the page, maybe it would be good to specify what they are prior to explaining why we store files into filestore.

Reply:
The point is that we are comparing native types (Balnce, bytecode storage and datastore), with object build on top of them (files, zip archives, websites). I think that we should separate them in the explanation and clarify that some are more complex objects build on top of simple native one.